includes/discovery/sensors/temperatures/f5.inc.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+if ($device['os'] == 'f5') {
+    $f5 = null;
+    // Get the Chassis Temperature values
+    //Pull the sysChassisTempTable table from the snmpwalk
+    $f5 = snmpwalk_cache_oid($device, 'sysChassisTempTable', array(), 'F5-BIGIP-SYSTEM-MIB');
+
+    if (is_array($f5)) {
+        echo "sysChassisTempTable ";
+
+        foreach (array_keys($f5) as $index) {
+            $descr           = "sysChassisTempTemperature.".$f5[$index]['sysChassisTempIndex'];
+            $current         = $f5[$index]['sysChassisTempTemperature'];
+            $sensorType      = 'f5';
+            $oid             = '.1.3.6.1.4.1.3375.2.1.3.2.3.2.1.2.'.$index;
+            $low_limit       =  null;
```

Comment:
This seems unnecessary to set all these variables when they are only ever null, just pass null to the function.